### PR TITLE
ctrlx-landing-page

### PR DESCRIPTION
**Pull request by MarianRaphael**
## Description

- A "hidden" landing page for ctrlX PLC users (no links from the main website to this page)
- Ability to submit support requests
- Option to "purchase" Node-RED support for ctrlX -> Forwarded to the contact form

### CODE REVIEW

**Comment by MarianRaphael** [comment]
@joepavitt my image src="images/partners/ctrlX-logo.png" doesn't seem to be optimized by the automation. Should I change the path?

File: src/partners/ctrlx.njk [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+                    <p class="mt-4">Need guidance on Node-RED? Seek support for your issues in the Community Forum, offering a plethora of
+                        resolutions.</p>
+                    <a class="ff-btn ff-btn--secondary uppercase" href="https://discourse.nodered.org/">Community Forum</a>
+                </div>
+            </div>
+            <div class="pricing-tile pt-4 pb-6 px-8">
+                <div class="flex flex-col justify-between items-center mb-4 sm:px-4 md:flex-row md:-mb-0 md:px-0 md:block">
+                    <h4>Professional Support</h4>
+                    <div>
+                        <h2>$15</h2>
```
Comment (ZJvandeWeg) [inline src/partners/ctrlx.njk:33]:
```suggestion
                        <label class="sub-label">from </label><h2>$15</h2>
```

**Review by ZJvandeWeg — APPROVED** [review]

**Comment by joepavitt** [comment]
@MarianRaphael that should have run. @Pezmc unintended consequence of eleventy changes, although thought this was part of git workflow, so not sure it wouldn't have triggered

**Comment by Pezmc** [comment]
@joepavitt The Git workflow is completely separate from the eleventy changes so very unlikely to be connected. 

The image pipeline will still resize this giant image down to 288px wide.

The workflow did run, but could not further optimize this PNG: https://github.com/flowforge/website/actions/runs/4850120654/jobs/8642743976

@MarianRaphael If you're unlikely to use the image at the now uploaded 9162 × 7189px, I'd suggest downsizing it to something more reasonable and opening a PR, say 1080x847px as this will improve build time and decrease the size of out git repo.

**Comment by MarianRaphael** [comment]
https://github.com/flowforge/website/pull/708

**Pull request merged**